ably.podspec

```diff
@@ -0,0 +1,30 @@
+#
+# Be sure to run `pod lib lint ably.podspec' to ensure this is a
+# valid spec and remove all comments before submitting the spec.
+#
+# Any lines starting with a # are optional, but encouraged
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = "ably"
+  s.version          = "1.0.2"
```

Comment:
We are not at version 1.0, our most recent release is 0.8, see  https://github.com/ably/ably-ruby/commit/a2d429aeebb9f87e7b43263bba24e6d5c75d24d6, however until we fully support the new token params we should be on 0.7.x